common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -472,8 +493,8 @@ public void onComplete(String streamId) {
           if (isStaleBlockOrTooLate) {
             // Throw an exception here so the block data is drained from channel and server
             // responds RpcFailure to the client.
-            throw new RuntimeException(String.format("Block %s %s", streamId,
-              ErrorHandler.BlockPushErrorHandler.TOO_LATE_OR_STALE_BLOCK_PUSH_MESSAGE_SUFFIX));
+            throw new RuntimeException("Block " + streamId + " " +
```

Comment:
Just a suggestion, may be we can explore if `MessageFormatter` from slf4j would work here or not as it is used in the logging framework so it must be performant enough and at the same time makes the code easy to read.

Reply:
Marking as resolved since we no longer format a string in the latest version.